app/models/competition.rb

```diff
@@ -637,7 +637,7 @@ def championship_warnings
     warnings
   end
 
-  def info_for(user)
```

Comment:
This one is interesting. The original purpose was to show "info for the given user". That's why the (pre-RuboCop) method kinda implies the name "info for user". I am not sure at which point passing in the user became unnecessary, but presumably it's because we're moving more and more warnings to React.

With all that being said, the method should probably be renamed if this argument is being removed, because it means now there is no user anymore that we're generating information _for_. So the "for" should not appear in the name.